sky/templates/aws-ray.yml.j2

```diff
@@ -97,19 +98,21 @@ setup_commands:
   # Line 'sudo grep ..': set the number of threads per process to unlimited to avoid ray job submit stucking issue when the number of running ray jobs increase.
   # Line 'mkdir -p ..': disable host key check
   # Line 'python3 -c ..': patch the buggy ray files and enable `-o allow_other` option for `goofys`
-  - mkdir -p ~/.ssh; touch ~/.ssh/config;
+  - sudo systemctl stop unattended-upgrades || true;
```

Comment:
Q: the user reported that "I was force killing the process with kill -9 ; without it, the systemctl stop also hangs for me".  Did we see the same problem (if so, this line will hang)?

Reply:
Yes, the line will hang if the `unattended-upgrades` process is running. We moved it to the front to take best effort to stop it before the process starts. Force killing the process does not work well, as when I manually tried to force killing it on the VM, a new `unattended-upgrades` will start immediately after killing the previous one.